FRENCH/src/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.md

```diff
@@ -0,0 +1,883 @@
+private `seasonal_fruit` field. This models the case in a restaurant where the
+customer can pick the type of bread that comes with a meal, but the chef
+decides which fruit accompanies the meal based on what’s in season and in
+stock. The available fruit changes quickly, so customers can’t choose the fruit
+or even see which fruit they’ll get.
+-->
+
+Nous pouvons aussi utiliser `pub` pour déclarer des structures et des
+énumérations publiquement, mais il y a d'autres points à prendre en compte. Si
+nous utilisons `pub` avant la définition d'une structure, nous rendons la
+structure publique, mais les champs de la structure restent privés. Nous pouvons
+rendre chaque champ public ou non au cas par cas. Dans l'encart 7-9, nous avons
+défini une structure publique `cuisines::PetitDejeuner` avec un champ public
+`tartine_grillee` mais avec un champ privé `fruit_de_saison`. Cela simule un
+restaurant où le client peut choisir le type de pain qui accompagne le repas,
+mais le chef décide des fruits qui accompagnent le repas en fonction de la
+saison et ce qu'il y a en stock. Les fruits disponibles changent rapidement,
+donc les clients ne peuvent pas choisir le fruit ou même voir quel fruit ils
+obtiendront.
+
+<!--
+<span class="filename">Filename: src/lib.rs</span>
+-->
+
+<span class="filename">Fichier : src/lib.rs</span>
+
+<!--
+```rust
+mod back_of_house {
+    pub struct Breakfast {
+        pub toast: String,
+        seasonal_fruit: String,
+    }
+
+    impl Breakfast {
+        pub fn summer(toast: &str) -> Breakfast {
+            Breakfast {
+                toast: String::from(toast),
+                seasonal_fruit: String::from("peaches"),
+            }
+        }
+    }
+}
+
+pub fn eat_at_restaurant() {
+    // Order a breakfast in the summer with Rye toast
+    let mut meal = back_of_house::Breakfast::summer("Rye");
+    // Change our mind about what bread we'd like
+    meal.toast = String::from("Wheat");
+    println!("I'd like {} toast please", meal.toast);
+
+    // The next line won't compile if we uncomment it; we're not allowed
+    // to see or modify the seasonal fruit that comes with the meal
+    // meal.seasonal_fruit = String::from("blueberries");
+}
+```
+-->
+
+```rust
+mod cuisines {
+    pub struct PetitDejeuner {
+        pub tartine_grillee: String,
+        fruit_de_saison: String,
+    }
+
+    impl PetitDejeuner {
+        pub fn en_ete(tartine_grillee: &str) -> PetitDejeuner {
+            PetitDejeuner {
+                tartine_grillee: String::from(tartine_grillee),
+                fruit_de_saison: String::from("pêches"),
+            }
+        }
+    }
+}
+
+pub fn manger_au_restaurant() {
+    // On commande un petit-déjeuner en été avec tartine grillée au seigle
+    let mut repas = cuisines::PetitDejeuner::en_ete("seigle");
+    // On change d'avis sur le pain que nous souhaitons
+    repas.PetitDejeuner = String::from("blé");
```

Comment:
```suggestion
    repas.tartine_grillée = String::from("blé");
```

Reply:
Wow ! Nice catch ! This is mainly why we need to proofread the translation !